src/components/interaction-designer/toolbar/ErrorNotifications.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <main class="error-notifications-wrapper">
+  <main class="error-notifications error-notifications-wrapper">
```

Comment:
we probably don't need `error-notifications-wrapper` either

Reply:
:heavy_check_mark: